LambdaEngine/Source/Game/ECS/Systems/Rendering/AnimationSystem.cpp

```diff
@@ -34,8 +36,9 @@ namespace LambdaEngine
 		}
 
 		// Call the graphs tick
-		animation.Graph.Tick(GetDeltaTimeInSeconds(), GetTotalTimeInSeconds(), skeleton);
+		animation.Graph.Tick(GetTotalTimeInSeconds(), skeleton);
 
+		// TODO: Remove this since it is only for testing
```

Comment:
Another TODO you might have forgotten to remove.

Reply:
Nope not forgotten since its still in testing phase